ansible/roles/infra-osp-template-generate/defaults/main.yaml

```diff
@@ -8,3 +8,71 @@ opentlc_admin_pub_keys:
 all_ssh_authorized_keys: "{{ opentlc_admin_pub_keys + [user_pub_key|d()] }}"
 
 provider_network: external
+
+default_security_groups:
```

Comment:
I'm confused how this is being used. In place of SGs defined for a config? Merged with it? Did you make changes to the Heat templates for this?

Reply:
yep, pushed now